scripts/bootstrap.sh

```diff
@@ -37,6 +38,11 @@ python-yaml"
 
 apt-get install -y $APT_PACKAGES
 
+# Install `lessc` dependencies for css pre-processing
+npm install -g less
+npm update -g less
```

Comment:
I think this is unnecessary; will remove in a followup which addresses i18n vagrant issues